tests/test_config.py

```diff
@@ -1,4 +1,4 @@
-# coding: utf-8
```

Comment:
What does this do?

Reply:
I'm guessing that would be me typing into the wrong window. :fearful:  Removing now.

Reply:
That's how you tell Python 2.x that the file you're in might contain UTF8 bytes.  By default, it parses files with ascii.  https://www.python.org/dev/peps/pep-0263/